### ISSUE TITLE
Clean up & comment code

### ISSUE THREAD

**Issue by nickmusi** [nickmusi/tonic-relation#8]

**Comment by hrshvrdhnsingh** [comment]
Hey! Could I Work on this issue and several others that have been raised? I think I can a few of them.

**Comment by nickmusi** [comment]
Yes, that would be great! Just fork and submit a pull request, and I will review and merge it!
